### DISCUSSION TITLE
Superset collapse a native filter by default on load

### DISCUSSION THREAD

**Discussion by SumeethKumar94** [apache/superset#28493] [Q&A / Help]
I have an apache superset dashboard with textboxes on click on markdown component it redirects to different dashboard with respective charts . I was able to use  _'&expand_filters=0'_ in URL to collapse filter when redirected to dashboard . 
Now we have native filter in main dashboard I am unable to collapse filter by default when dashboard is loaded is there any JSON Metadata keys or any other way I can collapse a filter by default ?

**Comment by Aalbedon** [comment]
I'm trying to find this option myself for the embedded dashboards that we use.
If I recall the option used to be there when you edited the properties of a dashboard.
The option made it possible to set the default behavior of the dashboard's filter panel without the need to supply a URL parameter.
